packages/quantic/force-app/main/default/lwc/quanticCategoryFacet/quanticCategoryFacet.html

```diff
@@ -1,34 +1,52 @@
+                                <lightning-button-icon variant="bare" icon-name="utility:chevronleft"></lightning-button-icon>
+                                <span class="nowrap">{labels.allCategories}</span>
+                           </div> 
+                       </li> 
+                    </template>
+                    <template for:each={nonActiveParents} for:item="v">
+                        <c-quantic-category-facet-parent onselectvalue={onSelect} key={v.value} item={v}></c-quantic-category-facet-parent>
+                    </template>
+                    <template if:true={activeParent}>
+                        <li class="slds-var-m-horizontal_medium slds-grid">
+                            <span class="nowrap facet-text__bold slds-var-m-horizontal_small">{activeParent.value} ({activeParent.numberOfResults})</span>
+                        </li>  
+                    </template>
+                    <li class=" slds-var-m-horizontal_medium facet-list_type slds-grid">
+                        <ul>
+                        <template for:each={values} for:item="v">
+                            <c-quantic-category-facet-value onselectvalue={onSelect} key={v.value} item={v}></c-quantic-category-facet-value>
+                        </template>
+                        </ul>
                     </li>
+                </ul>
+                <template if:true={canShowLess}>
+                    <button variant="base" label={labels.showLess}
+                      class="slds-var-m-left_x-small facet__show-less" onclick={showLess}>
+                      <span class="facet__show-less_text">- {labels.showLess}</span>
+                    </button>
                 </template>
-            </ul>
-            <ul class="slds-has-dividers_around-space slds-m-vertical_medium">
-                <template for:each={values} for:item="v">
-                    <c-quantic-category-facet-value onselectvalue={onSelect} key={v.value} item={v}></c-quantic-category-facet-value>
-                </template>
-            </ul>
-
-            <template if:true={canShowLess}>
-              <button variant="base" label={labels.showLess}
-                class="slds-m-left_x-small facet__show-less" onclick={showLess}>
-                <span class="facet__show-less_text">- {labels.showLess}</span>
-              </button>
-            </template>
-            <template if:true={canShowMore}>
-              <button variant="base" label={labels.showMore}
-                class="slds-m-left_x-small facet__show-more" onclick={showMore}>
-                <span class="facet__show-more_text">+ {labels.showMore}</span>
-              </button>
+                <template if:true={canShowMore}>
+                    <button variant="base" label={labels.showMore}
+                      class="slds-var-m-left_x-small facet__show-more" onclick={showMore}>
+                      <span class="facet__show-more_text">+ {labels.showMore}</span>
```

Comment:
I'd rename the CSS class name to  `facet__show-more-text` as `text` is not a modifier.